easybuild/easyconfigs/n/NanopolishComp/NanopolishComp-0.6.11-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'NanopolishComp'
+version = '0.6.11'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://adrienleger.com/NanopolishComp/'
+description = "NanopolishComp is a Python3 package for downstream analyses of Nanopolish output files"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('tqdm', '4.47.0'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    (name, version, {
+        'modulename': 'NanopolishComp',
+        'source_urls': ['https://pypi.python.org/packages/source/n/NanopolishComp'],
+        'checksums': ['753eed9ed93d12b15b693c43382c73379064d86eeef651611fa27abe7f8a44b3'],
+    }),
+]
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

Comment:
For PythonBundle and PythonPackage, this is the default sanity check already, so you can remove it